snorkel/labeling/model/baselines.py

```diff
@@ -1,8 +1,31 @@
 import numpy as np
+import torch
+from scipy.sparse import issparse
 
 from .label_model import LabelModel
 
 
+def to_numpy(Z):
```

Comment:
Smells that we have to_numpy() here and arraylike_to_numpy() in `analysis/utils.py`. Can they be combined?